easybuild/easyconfigs/r/ReFrame/ReFrame-3.0.eb

```diff
@@ -0,0 +1,93 @@
+easyblock = 'PythonPackage'
+
+name = 'ReFrame'
+version = '3.0'
+
+homepage = 'https://github.com/eth-cscs/reframe'
+description = '''ReFrame is a framework for writing regression tests for HPC systems.'''
+
+toolchain = SYSTEM
+
+
+sources = [{
+    'filename': 'v%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/eth-cscs',
```

Comment:
There is no need to use `git_config` when the source code is hosted on GitHub, just use:

```python
source_urls = ['https://github.com/eth-cscs/reframe/archive/']
sources = ['v%(version)s.tar.gz']
```

In fact, it's strongly recommended to use the suggested approach, since that way the checksum will always be the same (which is not the case when using `git_config` because the actual source tarball is created locally, see also https://github.com/easybuilders/easybuild-framework/issues/2726